apps/blog/src/posts/ios-iap-experiment/ios-iap-experiment.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: You Won’t Believe How We Enabled In-App Purchases for PWAs on iOS
+excerpt: This post guides us through the journey of how we wrangled Swift to integrate in-app purchases in iOS PWAs.
+description: A step-by-step account of integrating in-app purchases in iOS PWAs despite the restrictions.
+date: 2023-11-07
+updatedDate: 2023-11-03
+trending: true
+featured: true
+image: posts/ios-iap-experiment/pic-1-min.jpg
```

Comment:
Just double checking since this will be used as the title image for the preview card. Can you check the dementions [here](https://www.figma.com/file/MKOsyENr69zpM8qy0mDudY/Blogs?type=design&node-id=11-86&mode=design&t=f0mq8JZgSPC4kxTo-0) against the existing posts and make sure this image is the same size? If you need assistance, you can ask @sanadeee for help. Sana has created a "hero" version of an image of choice for the past posts.